chatwoot/chatwoot
# feat: Ability to send attachment in new conversation

# Pull Request Template

## Description

This PR allows sending attachments while creating a new conversation for the modal.

Fixes https://linear.app/chatwoot/issue/CW-2301/feat-ability-to-send-attachment-in-new-conversation

Reverted PR https://github.com/chatwoot/chatwoot/pull/7698

## Type of change


## How Has This Been Tested?

**Test cases**

1. Send messages from the new conversation modal with and without attachments.
2. Send messages through all inboxes except Whatsapp Cloud
3. Try to drag and drop through the conversation and contact pages.
4. Try attaching the through button from the modal, and try multiple attachments (manually and drag).
5. Check the reply box attachments flow (drag and drop)
6. Check drag and drop with the new conversation modal active in the conversation screen.

## Checklist:
